contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -501,14 +501,14 @@ contract NodeOperatorsRegistry is AragonApp, IStakingModule, Versioned {
     }
 
     /// @notice Requests the given number of the validator keys from the staking module
-    /// @param _keysCount Requested keys count to return
+    /// @param _depositsCount Requested keys count to return
     /// @return returnedKeysCount Actually returned keys count
     /// @return publicKeys Batch of the concatenated public validators keys
     /// @return signatures Batch of the concatenated signatures for returned public keys
-    function requestValidatorsKeysForDeposits(uint256 _keysCount, bytes)
+    function provideDepositsData(uint256 _depositsCount, bytes)
```

Comment:
`provide` implies that we're submitting something, whereas here we're requesting something instead, so I'd rename it to smth like `requestDepositData` or `obtainDepositData`

Reply:
Like `obtainDepositData` 👍🏻. Renamed.